chapters/operatorsandexpressions.tex

```diff
@@ -959,32 +959,35 @@ \subsubsection{spatialDistribution}\label{spatialdistribution}
 \begin{nonnormative}
 Many applications involve the modelling of variable-speed transport of properties.
 One option to model this infinite-dimensional system is to approximate it by an ODE, but this requires a large number of state variables and might introduce either numerical diffusion or numerical oscillations.
-Another option is to use a built-in operator that keeps track of the spatial distribution of $z(x, t)$, by suitable sampling, interpolation, and shifting of the stored distribution.
+Another option is to use a built-in operator that keeps track of the spatial distribution of $z(\xi, t)$, by suitable sampling, interpolation, and shifting of the stored distribution.
 In this case, the internal state of the operator is hidden from the ODE solver.
 \end{nonnormative}
 
 \lstinline!spatialDistribution! allows the infinite-dimensional problem below to be solved efficiently with good accuracy
 \begin{align*}
-\frac{\partial z(x,t)}{\partial t}+v(t)\frac{\partial z(x,t)}{\partial x} &= 0.0\\
-z(0.0, t) &= \mathrm{in}_0(t) \text{ if $v\geq 0$}\\
-z(1.0, t) &= \mathrm{in}_1(t) \text{ if $v<0$}
+\frac{\partial z(\xi, t)}{\partial t} + v(t) \frac{\partial z(\xi, t)}{\partial x} &= 0\\
```

Comment:
```suggestion
\frac{\partial z(\xi, t)}{\partial t} + v(t) \frac{\partial z(\xi, t)}{\partial \xi} &= 0\\
```
As indicated I'm not sure about the change from $x$ to $\xi$, but if done it should be done consistently.